Teensy/src/gpio/gpio.cpp

```diff
@@ -53,12 +57,42 @@ bool updateSensorInfo(sensorData_s *pSensorData)
     return false;
 }
 
+
 static WheelSpeed_s gpio_getWheelSpeed()
 {
     WheelSpeed_s wheelSpeed = {};
     return wheelSpeed;
 }
 
+    // setup for GPS readings
+    SoftwareSerial mySerial(GPS_RX_PIN, GPS_TX_PIN);
+    Adafruit_GPS GPS(&mySerial);
+    // connect at 115200 so we can read the GPS fast enough and echo without dropping chars
+    Serial.begin(115200);
+    delay(5000);
```

Comment:
We don't really want any delays ideally, so I testing we will look at trying to remove this